src/weapons.cpp

```diff
@@ -1,5 +1,6 @@
 
 #include "audio/audio.h"
+#include "controls.h"

```

Comment:
hmmm ok, I know how to do it without this include )

Reply:
Hi !
How could this have been done ?
I'm interested in OpenTomb internal architecture.

Reply:
done in my commit - just substract weapon code outside function, that uses it  ))